LetsBora/LetsBora/Screens/CreateEvent/Category/CategoryViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  CategoryViewController.swift
+//  LetsBora
+//
+//  Created by Davi Paiva on 06/06/25.
+//
+
+import UIKit
+
+class CategoryViewController: UIViewController {
+    
+    private var availableOptions: [Tag]?
+    private var initialOption: Tag?
+    private var selectedOption: Tag?
+    var onSelectCategory: ((Tag) -> Void)?
+    
+    init(
+        option: [Tag]? = nil,
```

Comment:
Como é um array, é melhor o nome da variável ser no plural
```suggestion
        options: [Tag]? = nil,
```